### ISSUE TITLE
Azure NetworkPolicy addons spec broken in 1.16

### ISSUE THREAD

**Issue by jackfrancis** [Azure/aks-engine#2069]
We aren't properly configuring the NPM addon to deal w/ the E2E validation:

```
[1mSTEP[0m: Ensuring that the correct resources have been applied for azure-npm
 
 [91m[1m•! Panic [101.022 seconds][0m
 Azure Container Cluster using the Kubernetes Orchestrator
 [90m/go/src/github.com/Azure/aks-engine/test/e2e/kubernetes/kubernetes_test.go:123[0m
   regardless of agent pool type
   [90m/go/src/github.com/Azure/aks-engine/test/e2e/kubernetes/kubernetes_test.go:124[0m
     [91m[1mshould have addons running [It][0m
     [90m/go/src/github.com/Azure/aks-engine/test/e2e/kubernetes/kubernetes_test.go:675[0m
 
     [91m[1mTest Panicked[0m
     [91mruntime error: index out of range[0m
     /usr/local/go/src/runtime/panic.go:44
 
     [91mFull Stack Trace[0m
     	/usr/local/go/src/runtime/panic.go:522 +0x1b5
     github.com/Azure/aks-engine/test/e2e/kubernetes.glob..func2.1.22()
     	/go/src/github.com/Azure/aks-engine/test/e2e/kubernetes/kubernetes_test.go:700 +0xdd8
     github.com/Azure/aks-engine/vendor/github.com/onsi/ginkgo/internal/leafnodes.(*runner).runSync(0xc000364a20, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
     	/go/src/github.com/Azure/aks-engine/test/e2e/kubernetes/kubernetes_suite_test.go:17 +0x10e
     testing.tRunner(0xc000107700, 0xaf8d10)
     	/usr/local/go/src/testing/testing.go:865 +0xc0
     created by testing.(*T).Run
     	/usr/local/go/src/testing/testing.go:916 +0x35a
```

**Comment by jackfrancis** [comment]
tl;dr here is that we are configuring 1.16 api models w/ an unnecessary container configuration, as the 1.16 implementation no longer includes the telemetry container
